en/_posts/2022-07-25-news-releases.md

```diff
@@ -0,0 +1,146 @@
+
+#### Up your alert-content-design game
+
+Placement of alerts is only half the battle. How you design them can make a world of difference in how effective they are.
+
+- **Reading level**: Write the alert text at a grade 6 to 8 reading level. You want people to easily understand why they need to pay attention.
+- **Length**: Keep alert text very minimal. Long alerts that include too much information are distracting and disorienting. This is especially true on mobile, where they push the main content down the screen. 
+- **Effective date**: Include the exact date that changes come into effect.
+- **Verb tense**: Pay attention to your verb tense. If changes don’t go into effect for another week, don’t write in the present tense. Put your alert text in the future tense and update it once the effective date has passed. 
+
+****
+
+<figure class="gc-complex-img" role="group">
+  <img alt="A long description can be found after the image."
+src="/images/Travel-alert-1.png" >
+</figure>
+<details>
+  <summary>What to do: Clear date, correct verb tense, repeated in-context within the page</summary>
+    <p>Long description: Alert text placed at the top of the page with a clear effective date:</p>
+    <p><b>Starting April 1, 2022</b>, pre-entry tests will no longer be required for fully vaccinated travellers entering Canada by land, air, or water. Until then, follow the pre-entry test requirements. You must still use ArriveCAN before and after April 1.</p>
+    <p>News release</p>
+</details>
+
+<figure class="gc-complex-img" role="group">
+  <img alt="A long description can be found after the image."
+src="/images/Travel-alert-2.png" >
+</figure>
+<details>
+  <summary>What to do: Clear date, correct verb tense, repeated in-context within the page</summary>
+    <p>Alert repeated within the relevant section of the page with clear effective date:</p>
+    <p><b>Starting April 1, 2022</b>, pre-entry tests will no longer be required for fully vaccinated travellers entering Canada by land, air, or water. Until then, follow the pre-entry test requirements. You must still use ArriveCAN before and after April 1.</p>
+    <p>News release</p>
+</details>
+
+
+<figure class="gc-complex-img" role="group">
+  <img alt="A long description can be found after the image."
+src="/images/Travel-alert-3.png" >
+</figure>
+<details>
+  <summary>What not to do: Critical date was missing and verbs are in present tense, so it made it seem like the rules were in effect already, (released February 15, rules were not changing until February 28)</summary>
+    <p>Example of alert text that is too vague and caused misunderstanding by using the wrong verb tense:</p>
+    <p><b>Adjustments to border measures</b></p>
+    <p>The Government of Canada lightens border measures as past of the transition of the pandemic response</p>
+    <p>News release</p>
+</details>

```

Comment:
```suggestion
<figure class="gc-complex-img" role="group">
  <img alt="A long description can be found after the image."
src="/images/Travel-alert-3.png" >
<figcaption>
  <details>
    <summary>What not to do: Critical date was missing and verbs are in present tense, so it made it seem like the rules were in effect already, (released February 15, rules were not changing until February 28)</summary>
      <p>Example of alert text that is too vague and caused misunderstanding by using the wrong verb tense:</p>
      <p><b>Adjustments to border measures</b></p>
      <p>The Government of Canada lightens border measures as past of the transition of the pandemic response</p>
      <p>News release</p>
  </details>
</figcaption>
</figure>
```

Reply:
We need to put the details/summary inside a figcaption tag inside the figure tag